### PR TITLE
Add Git for Windows icon

### LINKED ISSUE #10820: Request: Git for Windows
### Brand Name

Git Windows

### Website

https://git-scm.com/downloads/logos

### Popularity Metric
Ranked [226,499 globally on Similarweb](https://www.similarweb.com/website/gitforwindows.org/#overview), 8.1k+ stars at https://github.com/git-for-windows/git

### Official Resources for Icon and Color
[SVG](https://github.com/git-for-windows/git-for-windows.github.io/blob/db9a134ed0fd484568124c1f79cb011eddb9a21d/img/gwindows_logo.svg) available on GitHub.

### Additional Comments

input of svg needs to be put manually

### PR DESCRIPTION
**Pull request by anhthang**
image

**Issue:** closes #10820 

**Popularity metric:**

[GitHub Repo stars](https://github.com/git-for-windows/git)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6010,6 +6010,11 @@
             "hex": "212121",
             "source": "https://github.com/gitextensions/gitextensions/blob/273a0f6fd3e07858f837cdc19d50827871e32319/Logo/Artwork/git-extensions-logo.svg"
         },
+        {
+            "title": "Git for Windows",
+            "hex": "FF8080",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:6015]:
```suggestion
            "hex": "80B3FF",
```
Why this specific color?
I would lean on the blue as https://gitforwindows.org/ also uses the same on the web page. Plus it being git for windows. Just my opinion though. What do @simple-icons/pr-reviewers think?

**Review by PratyushJayachandran — COMMENTED** [review]
Icon matches perfectly though!

File: _data/simple-icons.json (hunk 1 above)
Comment (uncenter) [reply _data/simple-icons.json:6015]: I'd also agree with the blue - predominant color on the website (though technically a different shade) and blue *is* the primary and only color of the actual Windows logo (see https://commons.wikimedia.org/wiki/File:Windows_logo_and_wordmark_-_2021.svg).

Comment (anhthang) [reply _data/simple-icons.json]: I chose red for its similarity to the original Git icon, but you're right, blue might be a better fit for consistency with Windows conventions.

**Review by uncenter — APPROVED** [review]

**Pull request merged**